src/index.js

```diff
@@ -1,40 +1,96 @@
+/* eslint-disable */
```

Comment:
**[REQUIRED CHANGES]**

- Please, let's try to write clean codes that conform to the linter. The usage of linters would help us as developers to write better code. Please, kindly remove it and try to fix the linter error.